pkg/apis/workspaces/v1alpha2/components.go

```diff
@@ -54,6 +55,14 @@ type ComponentUnion struct {
 	// +optional
 	Container *ContainerComponent `json:"container,omitempty"`
 
+	// Allows importing into the devworkspace docker-compose files
+	// defined in a given manifest. For example this allows the reuse of previously
+	// docker-compose files used to define configuration for managing
+	// multiple containers at the same time.
+	//
+	// +optional
+	Compose *ComposeComponent `json:"compose,omitempty"`
```

Comment:
> I realize this follows the template of other component types, but I would drop mention of `devworkspace` in this description as it will be shown for devfiles as well and that may be confusing.

I am not sure if I understood your suggestion. Could you explain ?

> It's also unclear what we mean by:
> 
> > Allows importing [...] docker-compose files defined in a given manifest.
> > (what is a given manifest in this case?)

Yes, agreed. It's a bit unclear. Also, I shouldn't explicitly mention docker, as we would be following the compose-spec for this component.

Reply:
The documentation being edited here applies to both devfiles and DevWorkspaces. If the docs explicitly mention DevWorkspaces, then users using the devfile schema will see documentation that says
> Allows importing into the devworkspace docker-compose files. 

even though they are writing a Devfile.

Reply:
Yes @amisevsk it's a bit confusing for users.